raiden/raiden_service.py

```diff
@@ -675,14 +676,15 @@ def _start_alarm_task(self) -> None:
         self.alarm.start()
 
     def _start_rest_api(self) -> None:
-        self.api_server.start()
+        if self.api_server is not None:
```

Comment:
We usually use the "return early" approach to error handling. In this case
```python
if self.api_server is None:
    return
```
Also, is doing nothing the right thing to do in this case?